js/rpg_windows/Window_MenuStatus.js

```diff
@@ -43,7 +43,7 @@ Window_MenuStatus.prototype.numVisibleRows = function() {
 
 Window_MenuStatus.prototype.loadImages = function() {
     $gameParty.members().forEach(function(actor) {
-        ImageManager.loadFace(actor.faceName());
+        ImageManager.reserveFace(actor.faceName());
```

Comment:
Problem is not so simple.
Window_Status and Window_SkillStatus is used after Window_MenuStatus, So this is why original script works.

We'd better implement stack-based reservation attachment.